nativelib/src/main/scala/scala/scalanative/native/fcntl.scala

```diff
@@ -0,0 +1,39 @@
+package scala.scalanative
+package native
+
+import stat.mode_t
+
+@extern
+object fcntl {
```

Comment:
Please make sure that all the posix-related wrappers are in `scala.scalanative.posix`.